blindsidenetworks/scalelite
# GIT-781: Added env vars default and override params for create and join (closes #781)

## Description

This PR adds 4 new env variables: 
```
DEFAULT_CREATE_PARAMS
OVERRIDE_CREATE_PARAMS
DEFAULT_JOIN_PARAMS
OVERRIDE_JOIN_PARAMS
```
The `DEFAULT...` will only add the variable if it isn't already set, whereas the `OVERRIDE...` will force the param to equal whatever values are passed

## Testing Steps

1. Set any of the environment variables above using the syntax shown in the README
2. Send a create/join request
3. Confirm that the params are there as expected

## Screenshots (if appropriate):
